lib/iris/tests/unit/analysis/maths/__init__.py

```diff
@@ -201,12 +202,16 @@ def cube_func(self):
         # I.E. 'iris.analysis.maths.xx'.
         pass
 
-    def _test_partial_mask(self, in_place):
+    def _test_partial_mask(self, in_place, second_lazy=False):
         # Helper method for masked data tests.
         dat_a = ma.array([2.0, 2.0, 2.0, 2.0], mask=[1, 0, 1, 0])
         dat_b = ma.array([2.0, 2.0, 2.0, 2.0], mask=[1, 1, 0, 0])
 
-        cube_a = Cube(dat_a)
+        if second_lazy:
+            cube_a = Cube(da.from_array(dat_a))
+        else:
+            cube_a = Cube(dat_a)
+
         cube_b = Cube(dat_b)
```

Comment:
```suggestion
        if second_lazy:
            cube_b = Cube(da.from_array(dat_b))
        else:
            cube_b = Cube(dat_b)

        cube_a = Cube(dat_a)
```

Reply:
Have now switched a and b
https://github.com/SciTools/iris/pull/3790/commits/18e2ff8f22989f12dbe0a3b050a37819b64811d6